local/bin/2-upload-dependencies-to-pcf

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+# get the meta-buildpack and upload to cf
+curl -L $(curl -s https://api.github.com/repos/cf-platform-eng/meta-buildpack/releases/latest | jq -r ".zipball_url") > meta-buildpack.zip
+unzip meta-buildpack.zip
```

Comment:
this doesn't work, unfortunately; if you call `unzip meta-buildpack.zip -d meta-buildpack` it will create a `meta-buildpack` dir with a `cf-platform-eng-meta-buildpack-a2ae1d7` dir inside it that has the code.

Reply:
lol, that's silly. Okay.